crates/sui-framework/tests/coin_balance_tests.move

```diff
@@ -15,7 +15,8 @@ module sui::test_coin {
 
     #[test]
     fun type_morphing() {
-        let test = &mut test_scenario::begin(&@0x1);
+        let scenario = test_scenario::begin(@0x1);
+        let test = &mut scenario;
```

Comment:
My naming here tried to minimize changes. So scenario/test vs scenario_val/scenario usually depended on the prior state of the file. 

But I otherwise agree. It really bothered me while changing the tests :)